extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -131,6 +131,17 @@ oppia.factory('ItemSelectionInputValidationService', [
                         'please select only one answer choice.')
                     });
                   }
+                } else if (rule.type === 'IsProperSubsetOf') {
+                  handledAnswers[choiceIndex] = true;
+                  if (ruleInputs.length < 2) {
+                    warningsList.push({
```

Comment:
Actually -- this particular warning doesn't need maxAllowedCount to be 1, does it?

Reply:
@seanlip 
 It does not, It should warn If ruleInputs.length < 2, regardless of maxAllowedCount, I will add it in the code.
 Also sorry for delaying this PR, I had a lot of assignment deadlines, and most of my time was spent console logging variables inside the spec files to see how what works, I will be more active from now.